src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write code here
+const listOfEmployees = document.querySelector('ul');
+const employees = [...listOfEmployees.children];
+
+function sortList(list) {
+  function employeeSalary(employee) {
+    return employee.dataset.salary
```

Comment:
Remove this function outside of `sortList`. 
And transfer the value that you need to change there immediately, instead of getting it inside the function